package.json

```diff
@@ -233,7 +234,7 @@
   },
   "engines": {
     "node": "14.x",
-    "npm": "6.x"
+    "npm": ">=6.x"
```

Comment:
Is this change related to this? 🤔 Also note that changing this might be a bit problematic since this will probably allow users to use npm v7 which will change the `package-lock.json` as we discussed on Slack; https://opencollective.slack.com/archives/C0RMV6F8C/p1629743374124200